src/common/pages/ErrorPage.vue

```diff
@@ -5,10 +5,15 @@
                  src="/images/error-octos.gif"
             >
             <h2 class="error-code">
-                404
+                {{ statusCode }}
             </h2>
             <h3 class="error-message">
-                {{ $t('COMMON.ERROR.404_MSG') }}
+                <template v-if="statusCode === '403'">
+                    {{ $t('COMMON.ERROR.404_MSG') }}
+                </template>
+                <template v-else>
+                    {{ $t('COMMON.ERROR.404_MSG') }}
+                </template>
```

Comment:
question: Both cases seem to be same?

Reply:
Yes. This is just to distinguish the case for internal users and developers.
There is no planning. I just added it 😅

Reply:
Oh I see... 🫡